deep-learning/_posts/2019-05-09-PPO.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: review
+title:  "Clipped Proximal Policy Optimization Algorithm"
+tags:   deep-learning
+author: Antoine Théberge
+pdf:    "https://arxiv.org/pdf/1707.06347"
+cite:
+    authors: "John Schulman, Filip Wolski, Prafulla Dhariwal, Alec Radford, Oleg Klimov"
+    title:   "Proximal Policy Optimization Algorithms"
+    venue:   "Arxiv"
+---
+
+## Introduction
+
+PPO is currently a state of the art algorithm for reinforcement learning that builds on TRPO$$^{[1]}$$ and is said to have the stability of trust-region methods while being much simpler to implement, requiring only a few changes to vanilla policy gradient methods.
+
+The paper has two main contributions: the clipped surrogate objective and multiple epochs for policy updates. 
+
+While the paper also proposes another way of constraining the policy updates to a trust region by adding a KL penalty coefficient, this latter approach is both more complicated to understand and implement and has been shown empirically to produce worst results. It will therefore not be explained here.
+
+## Clipped Surrogate Objective
+
+Vanilla policy gradient methods work by optimizing the following loss
+
+![](/deep-learning/images/PPO/pg.jpg)
+
+where $$\hat{A}$$ is the advantage function. By performing gradient ascent on this, you will take actions that lead to higher rewards more likely. With the idea of importance sampling, we can replace the objective function with $$\hat{r}$$ = 
```

Comment:
$$\hat{r}$$ = 

is this a typo?

Reply:
No, it's to add a hat to the variable "r"